opensrp-chw-malaria/src/main/res/layout/activity_malaria_profile.xml

```diff
@@ -0,0 +1,158 @@
+                                                 android:layout_width="match_parent"
+                                                 android:layout_height="match_parent"
+                                                 android:background="@color/white">
+
+    <android.support.design.widget.AppBarLayout
+            android:id="@+id/collapsing_toolbar_appbarlayout"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:background="@color/light_gray">
+
+        <android.support.design.widget.CollapsingToolbarLayout
+                android:id="@+id/collapsing_toolbar_layout"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content">
+
+            <android.support.v7.widget.Toolbar
+                    android:id="@+id/collapsing_toolbar"
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    android:theme="@style/MalariaTheme.AppBarOverlay.Light"
+                    app:contentInsetEnd="0dp"
+                    app:contentInsetLeft="0dp"
+                    app:contentInsetRight="0dp"
+                    app:contentInsetStart="0dp"
+                    app:contentInsetStartWithNavigation="0dp"
+                    app:layout_collapseMode="pin">
+
+                <org.smartregister.view.customcontrols.CustomFontTextView
+                        android:id="@+id/toolbar_title"
+                        style="@style/CustomFontTextViewStyle.ClientList"
+                        android:layout_width="match_parent"
+                        android:layout_height="match_parent"
+                        android:gravity="center_vertical|start"
+                        android:paddingStart="0dp"
+                        android:paddingEnd="0dp"
+                        android:text="@string/return_to_profile"
+                        android:textColor="@color/customAppThemeBlue"
+                        android:textSize="17sp"/>
+            </android.support.v7.widget.Toolbar>
+
+            <LinearLayout
+                    android:id="@+id/profile_name_layout"
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    android:layout_marginTop="50dp"
+                    android:gravity="center_horizontal"
+                    android:orientation="vertical"
+                    android:paddingTop="10dp"
+                    app:layout_collapseMode="parallax">
+
+                <de.hdodenhof.circleimageview.CircleImageView
+                        android:id="@+id/imageview_profile"
+                        android:layout_width="80dp"
+                        android:layout_height="80dp"
+                        android:layout_gravity="center_horizontal"
+                        android:src="@mipmap/ic_child"
```

Comment:
I don't think we want to use to child Icon here

Reply:
What icon should i use?

Reply:
> What icon should i use?

Member Icon

Reply:
> > What icon should i use?
> 
> Member Icon

Also make sure that when the member has an image its displayed